packages/p2p/src/stores/general-store.js

```diff
@@ -46,6 +46,11 @@ export default class GeneralStore extends BaseStore {
         return this.props?.client || {};
     }
 
+    @computed
+    get setCurrentFocus() {
+        return this.props?.setCurrentFocus;
+    }
+
```

Comment:
```suggestion
```
Can we use the `setCurrentFocus()` in `ui-store` instead of declaring our own? There's a nice handling for on-screen keyboard if we use the `ui-store` instead of this